pkgs/development/tools/build-managers/cmake/002-application-services.diff

```diff
@@ -5,8 +5,8 @@ index 9a18184fd3..278d146dd1 100644
 @@ -933,7 +933,6 @@ endif()
  # On Apple we need CoreFoundation and CoreServices
  if(APPLE)
-   target_link_libraries(CMakeLib "-framework CoreFoundation")
--  target_link_libraries(CMakeLib "-framework CoreServices")
+   target_link_libraries(CMakeLib PUBLIC "-framework CoreFoundation")
+-  target_link_libraries(CMakeLib PUBLIC "-framework CoreServices")
```

Comment:
The patch needed to be changed because of this commit: https://github.com/Kitware/CMake/commit/9a24c648ebdc50495bf113a34373d0372bda125b

They just updated the usage syntax for target_link_libraries with the specifier.